packages/@aws-cdk/aws-apigatewayv2/README.md

```diff
@@ -204,6 +206,14 @@ const api = new apigwv2.HttpApi(this, 'HttpProxyProdApi', {
 });
 ```
 
+The properties certificate, endpointType, securityPolicy, certificateName, and ownershipVerificationCertificate are mapped in CFN to a DomainNameConfiguration.
+See [DomainNameConfiguration](https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-properties-apigatewayv2-domainname-domainnameconfiguration.html)
+for more details.
+You need only one DomainNameConfiguration, unless you're migrating from one endpoint type to another. To migrate to a new endpoint type,
+you add a new domain name configuration via `addDomainNameConfiguration` and then configure DNS records to route traffic to the new endpoint.
```

Comment:
Does this mean that one cannot have two "domain name configuration" with the same endpoint type? 
Is this a validation we should add?

Reply:
Consequently, should this method instead be called `addEndpointType()`?

Reply:
Yes, each configuration should have a unique endpoint. The validation is in our service code. We're not just adding an endpoint type in that method. The customer can choose to have a different tls policy, certificate will also be different for the new config. So renaming it to `addEndpointType()` might be confusing as it might mean that we're only allowing changing the `endpointType` property, which is not correct.

Reply:
Fair enough. How about `addEndpoint()` then?